web_src/less/modules/tippy.less

```diff
@@ -0,0 +1,82 @@
+.tippy-box[data-animation="fade"][data-state="hidden"] {
```

Comment:
Should this only happen when `[data-animation="fade"]`, seems like it should be independent of the animation.

Reply:
Some of these styles come from [tippy.js's suggested CSS](https://github.com/atomiks/tippyjs/blob/master/src/scss/animations/fade.scss), this is one of them.

Reply:
In fact, this rule and the `transition-property` below enable the `fade` animation. Not that I like it but I guess we could just leave it in.

Reply:
I think there could be more comments about where the styles come from and what they are for.

Reply:
Comment on style source added.